src/extension.ts

```diff
@@ -984,6 +1002,13 @@ export async function activate(context: vscode.ExtensionContext) {
         }
     );
 
+    const getIsPreviewMode = (): boolean => {
```

Comment:
It seems that getting started for CLUE is not hidden from the flag. Is that by design?

Reply:
That's a good catch! I'm not sure if it's necessary to hide it. @vandyliu  what do you think? 

We could even put a text that displays turn on preview mode to have access to this new simulator! :)

Reply:
I knew I forgot something. I'll add it. Thanks David :D

Reply:
I did your suggestion Kevin. Let me know what you think.

Reply:
I think it might interest a lot of people this way without having the risk of showing an incomplete device.